poppy/poppy_core.py

```diff
@@ -351,13 +352,17 @@ def display(self,what='intensity', nrows=1,row=1,showpadding=False,imagecrop=Non
             Display colorbar
         ax : matplotlib Axes
             axes to display into
+        use_angular_coordinates : bool, optional
+            Should the axes be labeled in angular units of arcseconds?
+            This is used by FresnelWavefront, where non-angular
+            coordinates are possible everywhere. Behavior for Wavefront
+            is undefined.
```

Comment:
The comment 'Behavior for Wavefront is undefined' is sort of cryptic. I wonder if it could be phrased more clearly as some variant of 'For non-Fresnel Wavefront, you don't need to worry about this.'

I suppose adding a new function parameter is less of a kludge than checking for a subclass attribute, yes.